src/EnergyPlus/DaylightingManager.cc

```diff
@@ -9984,29 +9980,33 @@ void ReportIllumMap(EnergyPlusData &state, int const MapNum)
 
         state.dataDaylightingData->IllumMap(MapNum).Name =
             format("{} at {:.2R}m", state.dataDaylightingData->IllumMap(MapNum).Name, state.dataDaylightingData->IllumMap(MapNum).Z);
-
-        for (R = 1; R <= state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).TotalDaylRefPoints; ++R) {
-            RefPts(state.dataDaylightingData->IllumMap(MapNum).Zone, R) =
-                format("RefPt{}=({:.2R}:{:.2R}:{:.2R})",
-                       R,
-                       state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).DaylRefPtAbsCoord(1, R),
-                       state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).DaylRefPtAbsCoord(2, R),
-                       state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).DaylRefPtAbsCoord(3, R));
-        }
     }
     if (SavedMnDy(MapNum) != state.dataEnvrn->CurMnDyHr.substr(0, 5)) {
         EnvrnPrint(MapNum) = true;
         SavedMnDy(MapNum) = state.dataEnvrn->CurMnDyHr.substr(0, 5);
     }
+
+    state.dataDaylightingData->IllumMap(MapNum).pointsHeader = "";
+    for (R = 1; R <= state.dataDaylightingData->ZoneDaylight(state.dataDaylightingData->IllumMap(MapNum).Zone).TotalDaylRefPoints; ++R) {
+        state.dataDaylightingData->IllumMap(MapNum).pointsHeader +=
+            format("RefPt{}=({:.2R}:{:.2R}:{:.2R})",
```

Comment:
I would add a space before Refpt to make the result a little more readable. `format(" RefPt . . . `
Current output is:
`Date/Time,WEST ZONE:CHICAGO_IL_USA ANNUAL HEATING 99% DESIGN CONDITIONS DB:MAP1 at 0.00m Illuminance [lux] (Hourly),RefPt1=(3.05:3.05:0.90),RefPt2=(3.05:3.05:0.00),RefPt3=(1.05:3.05:0.90),RefPt4=(3.05:1.05:0.00),,`

Also note there are two trailing commas for some reason. I looked at `pointsHeader` in the debugger, and the trailing commas are not here, so they're getting added later somewhere.
Oh, wait, there's a comment from long ago `// must add correct number of commas at end`. Ok, whatever.
Its odd that the z-coordinate of the map gets written to the sql output, but not to the map file. But looking back in the archive, it's always been that way. So, whatever.

So, the only change requested here is to add a space to " RefPt".